### PR TITLE
x64 fixes for issue #10

### PR DESCRIPTION
**Pull request by l0calh05t**

### CODE REVIEW

File: src/mspdb.cpp [hunk 1]
```diff
@@ -75,19 +79,27 @@ bool tryLoadMsPdb(const char* version, const char* mspdb, const char* path = 0)
 	return modMsPdb != 0;
 }
 
+#ifdef _M_X64
+#define MSPDB_DIR_NEW "..\\..\\VC\\bin\\amd64\\"
+#define MSPDB_DIR_OLD MSPDB_DIR_NEW
+#else
+#define MSPDB_DIR_NEW "..\\..\\VC\\bin\\"
+#define MSPDB_DIR_OLD 0
+#endif
```
Comment (rainers) [inline src/mspdb.cpp]: I'm not so happy with names `..._OLD` and `..._NEW`. Better use something descriptive, e.g. `BIN_DIR_LT_VS12` and `BIN_DIR_GE_VS12`.

File: src/mspdb.cpp [hunk 2]
```diff
@@ -51,7 +51,11 @@ bool getInstallDir(const char* version, char* installDir, DWORD size)
 	strcat(key, version);
 
 	HKEY hkey;
-	if (RegOpenKeyExA(HKEY_LOCAL_MACHINE, key, 0, KEY_QUERY_VALUE, &hkey) != ERROR_SUCCESS)
+	if (RegOpenKeyExA(HKEY_LOCAL_MACHINE, key, 0, KEY_QUERY_VALUE
+#if _M_X64
+		| KEY_WOW64_32KEY
+#endif
```
Comment (rainers) [inline src/mspdb.cpp]: I don't like slicing an expression with preprocessor directives these days. Instead recommend defining some `KEY_OPEN_FLAGS` somewhere before that includes `KEY_WOW64_32KEY` for x64 builds.

**Comment by rainers** [comment]
Other than nitpicks, looks good to me.

**Comment by l0calh05t** [comment]
Ok, I adjusted the style to match your preferences

**Comment by rainers** [comment]
Thanks.

**Comment by rainers** [comment]
I have removed the "Minimum Required Version" option from the x64 configurations, it is producing a warning in VS2013 and changes it to 6.0. The default of 5.02 is now used.

**Comment by l0calh05t** [comment]
Ah, I had fixed that as well but apparently didn't commit it by accident, thanks!

**Pull request merged**